src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -0,0 +1,168 @@
+package teammates.ui.controller;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+
+import teammates.common.datatransfer.CourseAttributes;
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.datatransfer.FeedbackSessionType;
+import teammates.common.datatransfer.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.EmailType;
+import teammates.common.util.Sanitizer;
+import teammates.common.util.TimeHelper;
+
+public abstract class InstructorFeedbackAbstractAction extends Action {
+
+    /**
+     * Common method to Get the feedback data
+     */
+    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(FeedbackSessionAttributes newSession) {
+
+        return this.extractFeedbackSessionDataHelper(newSession);
+    }
+
+    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(
+            FeedbackSessionAttributes newSession, List<String> sendRemainderEmailsList) {
+        return this.extractFeedbackSessionDataHelper(newSession, sendRemainderEmailsList);
```

Comment:
>See if you can combine them into one method.

I tried to put all of them in a single method but some of tests where failing and @wkurniawan07  said Refactoring should not modify the action's behavior i have added 2 methods instead of one method.